ros_ign_gazebo_demos/launch/air_pressure.launch.py

```diff
@@ -26,47 +26,34 @@
 from launch.substitutions import LaunchConfiguration
 from launch_ros.actions import Node
 
-# TODO; execution broken
-# Failed to create a bridge for topic [/air_pressure] with ROS2 type
-# [sensor_msgs/FluidPressure] and Ignition Transport type
-# [ignition.msgs.FluidPressure]
 
 def generate_launch_description():
 
     pkg_ros_ign_gazebo_demos = get_package_share_directory('ros_ign_gazebo_demos')
+    pkg_ros_ign_gazebo = get_package_share_directory('ros_ign_gazebo')
 
-    # Ignition Gazebo
-    ign_gazebo = IncludeLaunchDescription(
-        PythonLaunchDescriptionSource(
-            os.path.join(pkg_ros_ign_gazebo_demos, 'launch', 'ign_gazebo.launch.py'),
-        )
+    bridge = Node(
+        package='ros_ign_bridge',
+        node_executable='parameter_bridge',
+        arguments=['/air_pressure@sensor_msgs/msg/FluidPressure@ignition.msgs.FluidPressure'],
+        output='screen'
```

Comment:
This doesn't work yet, right? See https://github.com/ignitionrobotics/ros_ign/issues/78